smartconfig/entry.py

```diff
@@ -85,9 +68,9 @@ class ConfigEntry(metaclass=_ConfigEntryMeta):
     @classmethod
     def _register_entry(cls) -> None:
         """Set the `_path` attribute and register the entry."""
-        cls._path = cls._path_override or inspect.getmodule(cls).__name__
+        cls._path = cls._path_override or cls.__module__.__name__
```

Comment:
`__module__` is already a string. Don't use `__name__ `.

Reply:
I believe you are looking at an older commit, this has already been solved :)